org.eclipse.transformer.jakarta/src/main/resources/org/eclipse/transformer/jakarta/jakarta-renames.properties

```diff
@@ -156,3 +156,35 @@ javax.xml.ws.spi.http=jakarta.xml.ws.spi.http
 javax.xml.ws.spi=jakarta.xml.ws.spi
 javax.xml.ws.wsaddressing=jakarta.xml.ws.wsaddressing
 javax.xml.ws=jakarta.xml.ws
+
+# Servlet Properties
```

Comment:
I don't think property transform rules should be in jakarta-renames which is for package transforms. I think these should instead be in jakarta-direct which is about string constant transforms. Can you please move them to jakarta-direct?

Reply:
They are already exists in jakarta-direct so class file is transformed properly.
but, java file is not converted.

Reply:
OK, when #186 is fixed by #283, then direct rules will be applied to Java files. So I don't want to duplicate the direct rules in the rename rules.